parser.go

```diff
@@ -981,6 +981,50 @@ func (p *parser) parseResponseComment(pkgPath, pkgName string, operation *Operat
 	return nil
 }
 
+func (p *parser) getOperation(route string, method string) *OperationObject {
+	switch strings.ToUpper(method) {
+	case http.MethodGet:
+		return p.OpenAPI.Paths[route].Get
+	case http.MethodPost:
+		return p.OpenAPI.Paths[route].Post
+	case http.MethodPatch:
+		return p.OpenAPI.Paths[route].Patch
+	case http.MethodPut:
+		return p.OpenAPI.Paths[route].Put
+	case http.MethodDelete:
+		return p.OpenAPI.Paths[route].Delete
+	case http.MethodOptions:
+		return p.OpenAPI.Paths[route].Options
+	case http.MethodHead:
+		return p.OpenAPI.Paths[route].Head
+	case http.MethodTrace:
+		return p.OpenAPI.Paths[route].Trace
+	}
+	return nil
+}
+
+func (p *parser) setOperation(route string, method string, operation *OperationObject) error {
```

Comment:
I'm not seeing how this method could return an error? The only return I see is the `return nil`. Getters/setters aren't as common in Go but might be valid here.

Reply:
I can't either, just wanted to follow what I presumed was a convention. I think I can just omit the return type and return nothing - would that be preferable?

As for creating a getter & setter I just thought it was clearer having the two switch statements near each other. But if you prefer I'm happy to leave the "setter" code where it was in `parseRouteComment` and change the "getter" to a function that just does the `nil` check and returns a boolean. That might be clearer in terms of intent.

Reply:
Updated and added test 👍